Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -116,47 +135,47 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
   obj.AddAction("Star",
                 _("Star"),
                 _("Draw a star on screen"),
-                _("Draw at _PARAM1_;_PARAM2_ a star with _PARAM3_ points and radius: _PARAM4_ (inner radius: _PARAM5_, rotation: _PARAM6_ ) "
+                _("Draw at _PARAM1_;_PARAM2_ a star with _PARAM3_ points and radius: _PARAM4_ (inner radius: _PARAM5_, rotation: _PARAM6_) "
                 "with _PARAM0_"),
                 _("Drawing"),
                 "res/actions/star24.png",
                 "res/actions/star.png")
 
       .AddParameter("object", _("Shape Painter object"), "Drawer")
-      .AddParameter("expression", _("X coordinate of the center of the star"))
-      .AddParameter("expression", _("Y coordinate of the center of the star"))
+      .AddParameter("expression", _("X position of center"))
+      .AddParameter("expression", _("X position of center"))
       .AddParameter("expression", _("Number of points of the star (minimum: 2)"))
       .AddParameter("expression", _("Radius (in pixels)"))
       .AddParameter("expression", _("Inner radius (in pixels, half radius by default)"))
       .AddParameter("expression", _("Rotation (in degrees)"))
       .SetFunctionName("DrawStar")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
-  // These actions are not exposed yet as the way they work is unsure. See https://github.com/4ian/GDevelop/pull/1256
-/*
+
   obj.AddAction("Arc",
                 _("Arc"),
                 _("Draw an arc on screen"),
-                _("Draw at _PARAM1_;_PARAM2_ an arc with _PARAM3_ of radius, start angle: _PARAM4_, end angle: _PARAM5_ (anticlockwise: _PARAM6_) "
+                _("Draw at _PARAM1_;_PARAM2_ an arc with _PARAM3_ of radius, start angle: _PARAM4_, end angle: _PARAM5_ (anticlockwise: _PARAM6_, close path: _PARAM7_) "
                 "with _PARAM0_"),
                 _("Drawing"),
                 "res/actions/arc24.png",
                 "res/actions/arc.png")
 
       .AddParameter("object", _("Shape Painter object"), "Drawer")
-      .AddParameter("expression", _("X coordinate of the center of the arc"))
-      .AddParameter("expression", _("Y coordinate of the center of the arc"))
+      .AddParameter("expression", _("X position of center"))
+      .AddParameter("expression", _("Y position of center"))
       .AddParameter("expression", _("Radius (in pixels)"))
-      .AddParameter("expression", _("Start angle of the arc"))
-      .AddParameter("expression", _("End angle of the arc"))
+      .AddParameter("expression", _("Start angle of the arc (in degrees)"))
+      .AddParameter("expression", _("End angle of the arc (in degrees)"))
       .AddParameter("yesorno", _("Anticlockwise"))
+      .AddParameter("yesorno", _("Close path"))
```

Comment:
I wonder if we should be a bit more explicit: "Close the path with a line between start and end"?

Reply:
In the description yes.

Reply:
Ok, in this case let's go for:

`Draw an arc on screen. If "Close path" is set to yes, a line will be drawn between the start and end point of the arc, closing the shape.`